registry.k8s.io/MANUALLY_SYNCING_BUCKETS.md

```diff
@@ -0,0 +1,83 @@
+# Manually syncing buckets
+
+## Background
+
+In the CNCF AWS accounts, there are two accounts of concern:
+
+- _cncf/kubernetes/k8s-infra-accounts_
+- _cncf/kubernetes/registry.k8s.io/registry.k8s.io_admin_
+
+Using an IAM user inside of the k8s-infra-accounts account, it is possible to write to the registry.k8s.io mirror buckets.
+
+## Logging in
+
+Log in as an IAM user, which has the ability to assume the _registry.k8s.io_s3writer_ role:
+
+```bash
+aws configure
+```
+
+## Assuming roles
+
+In order to gain permissions to write to the buckets, it is required to call STS to assume the _registry.k8s.io_s3writer_ role:
+
+```bash
+unset AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN
+JSON=$(aws sts assume-role \
+  --role-arn "arn:aws:iam::513428760722:role/registry.k8s.io_s3writer"  \
+  --role-session-name registry.k8s.io_s3writer \
```

Comment:
My usual practice: set your role session name to include or match your IAM username.

Reply:
For example, my role session name when I run tools as _me_ is usually either my email address, or the string `tim`.

We also set constraints on the role so that I'm not allowed to set an invalid role session name.